sqlalchemy/ormDemo.py

```diff
@@ -0,0 +1,120 @@
+# Example SQLAlchemy ORM program that uses the Ingres dialect.
+#
+# The code creates table "Employees" (drops/recreates if exists),
+#  adds rows to the table, updates a row, deletes a row,
+#  queries and prints the rows after each operation.
+#
+# Environment variables:
+#   SQLALCHEMY_INGRES_ODBC_DRIVER_NAME - Name of Ingres ODBC driver, required
+#   DEMO_DB - Name of database to use, required
+#   DEMO_ID - Userid for connecting, optional
+#   DEMO_PW - Password for connecting, optional
+#   Note: DEMO_ID and DEMO_PW will only be used if both are set.
+#
+# Code is partly modeled after the tutorial in video:
+#   "SQLAlchemy Turns Python Objects Into Database Entries"
+#    by NeuralNine at:  https://www.youtube.com/watch?v=AKQ3XEDI9Mw
+#
+# Code tested with these Python packages:
+#   SQLAlchemy        2.0.25
+#   ingres_sa_dialect 0.4
+#   pypyodbc          1.3.6
+#   setuptools        63.2.0
+
+import os, sys
+from sqlalchemy import create_engine, ForeignKey, Column, String, Integer, CHAR, update, delete
+from sqlalchemy.orm import Session, declarative_base
+
+def get_connection_url():
+
+    if "DEMO_DB" in os.environ:
+        db = os.getenv("DEMO_DB")
+    else:
+        print("Please set environment variable DEMO_DB to the name of an existing database.")
+        sys.exit()
```

Comment:
exit(1)?

Alternative:

    raise NotImplementedError("Please set environment variable DEMO_DB to the name of an existing database.")

Reply:
Per https://docs.python.org/3/library/sys.html#sys.exit, changed the `else` path to simply:

`sys.exit("Please set environment variable DEMO_DB to the name of an existing database.")`